yt_dlp/extractor/gofile.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    try_get
+)
+
+
+class GofileIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?gofile\.io/d/(?P<id>[^/]+)'
+    _TESTS = [{
```

Comment:
This format of tests is not correct AFAIK. You are returning a playlist but checking for a file. See 4channel on how to write tests for such cases.